contracts/settings/platform/libraries/PlatformSettingsLib.sol

```diff
@@ -27,6 +27,14 @@ library PlatformSettingsLib {
         return AppStorageLib.store().platformSettings[name];
```

Comment:
what does `s` mean in this context. wouldn't it be better to name the function `store` or something more descriptive?

Reply:
just shorthand, but yes `store` is more descriptive. some lines of code started to get pretty long so we started doing this and just stuck with it.